tests/possible_values.rs

```diff
@@ -249,6 +265,42 @@ fn escaped_possible_values_output() {
     ));
 }
 
+#[test]
+fn alias() {
+    let m = App::new("pv")
+        .arg(
+            Arg::new("option")
+                .short('o')
+                .long("--option")
+                .takes_value(true)
+                .possible_value(ArgValue::new("test123").alias("123"))
+                .possible_value("test321")
+                .case_insensitive(true),
+        )
+        .try_get_matches_from(vec!["pv", "--option", "123"]);
+
+    assert!(m.is_ok());
+    assert!(m.unwrap().value_of("option").unwrap().eq("123"));
```

Comment:
Not sure why this and the below tests are needed. Is it for documenting that the aliases wouldn't change the given value?